validation/src/main/resources/application.conf

```diff
@@ -1,2 +1,14 @@
-hmda.validation.quality.Q005.loan.amount=1203
-hmda.validation.quality.Q005.loan.amount=${?Q005_LOAN_AMOUNT}
+hmda.validation.quality {
+  Q005 {
+    loan.amount=1203
+    loan.amount=${?Q005_LOAN_AMOUNT}
+  }
+  Q014 {
+    applicant.max_income=3000
```

Comment:
This should probably be switch to `max-income`, to match the pattern in the other `application.conf` files.